processor/src/stack/mod.rs

```diff
@@ -66,6 +66,32 @@ impl Stack {
         Ok(self.trace[0][self.step])
     }
 
+    /// Get states from the stack, which includes both the
+    /// trace state and overflow table.
+    /// If n is not passed in, this returns all states.
```

Comment:
nit: Would say "Returns" instead of "Get". Also, I usually break lines after 100 characters.